client/web/src/enterprise/batches/create/RunServerSideModal.module.scss

```
@@ -0,0 +1,66 @@
+.modal {
+    width: 55rem;
+    height: 25rem;
+}
+
+.content {
+    display: flex;
+    height: 95%;
+    justify-content: space-between;
+}
+
+.close {
+    display: flex;
+    position: absolute;
+    padding: 0;
+    top: 1rem;
+    right: 1rem;
+    border: none;
+    background: transparent;
+}
+
+.icon {
+    height: 1.125rem;
+    width: 1.125rem;
+    color: var(--icon-color);
+}
+
+.left {
+    display: flex;
+    flex-direction: column;
+    width: 72%;
+}
+
+.video-container {
+    height: 85%;
+    background-color: var(--body-bg);
+}
+
+.right {
+    margin-left: 1.5rem;
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+}
+
+.right-top {
+    background-color: var(--body-bg);
+    border: 1px solid var(--border-color);
+    padding: 0.8rem;
+    margin-bottom: 0.8rem;
```

On Figma, the padding and margin look a bit bigger here, and we usually use multiples of 0.125rem. I think we might want:

```suggestion
    padding: 1rem;
    margin-bottom: 1.5rem;
```